pkg/validating-webhooks/hiveextension/v1beta1/agentclusterinstall_admission_hook.go

```diff
@@ -250,6 +313,13 @@ func areImageSetRefsEqual(imageSetRef1 *hivev1.ClusterImageSetReference, imageSe
 	}
 }
 
+func isUserManagedNetworkingSetToFalseWithSNO(newObject *hiveext.AgentClusterInstall) bool {
+	return newObject.Spec.ProvisionRequirements.ControlPlaneAgents == 1 &&
```

Comment:
if you are adding mutation webhook then `UserManagedNetworking` can be nil it cannot be false

Reply:
To Sum up:
1) With SNO: the `UserManagedNetworking` field is not set, the mutation hook (which will be implemented on a separate PR) will set it to true. The validation then will be successful
2) With SNO, if the user specifically set the `UserManagedNetworking` field to false despite the documentation, the validation will fail the transaction (see the code above)
3) For multi node: if the user does not set the field the mutation hook will set it to false (its default value). For all the cases of multi node the above validation should succeed.

The mutation web hook as I mentioned will be implemented in a separate PR

Reply:
if the field is nil then the function `isUserManagedNetworkingSetToFalseWithSNO` returns false the validation moves to the next check